src/twisted/web/_http2.py

```diff
@@ -103,11 +103,23 @@ class H2Connection(Protocol, TimeoutMixin):
     @ivar _sender: A handle to the data-sending loop, allowing it to be
         terminated if needed.
     @type _sender: L{twisted.internet.task.LoopingCall}
+
+    @ivar abortTimeout: The number of seconds to wait after we attempt to shut
+    the transport down cleanly to give up and forcibly terminate it. This
```

Comment:
Not a blocker, but, to nitpick here, it would be nice if the indentation were consistent with the surrounding parameters (including the one you added :))